app/src/main/java/com/example/mycleaninglog/ExpandableCard.kt

```diff
@@ -236,86 +233,5 @@ viewModel.listenToMyRooms()
     }
 }
 
-fun saveItem(roomName: String, roomID: String,viewModel: MainViewModel ){
```

Comment:
Good suggestion, but this was already covered in a previous code review.  When that occurs, first in wins.